include/lorawan/lorawan.h

```diff
@@ -99,6 +99,20 @@ struct lorawan_join_config {
 	enum lorawan_act_type mode;
 };
 
+/**
+ * @brief Add battery level callback function.
+ *
+ * Provide the LoRaWAN stack with a function to be called whenever a battery
+ * level needs to be read. The function needs to return a value between 0-254
+ * for battery levels and/or 255 when unable to read the battery or no battery
+ * present.
```

Comment:
As per the API requirement of loramac-node `0: node is connected to an external power source, 1..254: battery level, where 1 is the minimum and 254 is the maximum value, 255: the node was not able to measure the battery level`

So why do we want to change the behaviour here?

Reply:
Fair. Missed the 0 part. Fixed now.